packages/postcss-merge-longhand/src/lib/canMerge.js

```diff
@@ -2,20 +2,17 @@ import isCustomProp from './isCustomProp';
 
 const important = node => node.important;
 const unimportant = node => !node.important;
-const hasInherit = node => ~node.value.indexOf('inherit');
-const hasInitial = node => ~node.value.indexOf('initial');
-const hasUnset = node => ~node.value.indexOf('unset');
 
-export default (props, includeCustomProps = true) => {
-    if (props.some(hasInherit) && !props.every(hasInherit)) {
-        return false;
-    }
+function hasIncompleteReservedKeyword (props) {
```

Comment:
I feel that this change damaged the readability of the function. It was definitely more clear what it did before.

Reply:
Fair enough. Happy to revert this, again I was just following the patterns in other parts of this codebase.